features/FEATURE_PSA/supporting_psa_in_mbed-os.md

```diff
@@ -40,8 +60,8 @@ Example:
     }
 ```
 
-### Adding single v7-M targets
-The PSA on single v7-M is supported with the help of Mbed OS PSA services. The following is taken from `targets/targets.json` and shows a PSA enabled target, `K64F`.
+## Adding single-core PSA targets
+A target can be categorized as a single-core target if it only has a single core. Mbed OS's PSA service emulation provides PSA compatibility for single-core PSA targets. The following is taken from `targets/targets.json` and shows a PSA-enabled single-core target, `K64F`.
```

Comment:
"A target can be categorized as a single-core target if it only has a single core." seems like this would be self-evident. We could remove this sentence.